public_html/application/views/template/homepage.php

```diff
@@ -148,6 +148,9 @@ function gtag(){dataLayer.push(arguments);}
 					<li>
 					<?= html::anchor('character/change_language/gr_GR', html::image(array('src' => 'media/images/flags-lang/gr.png'), array('title' => 'Greek', 'class' => 'img-responsive', 'alt' => 'Greece flag')));?>
 					</li>
+					<li>
+				        <?=html::ancor('character/change_language/es_ES', html::image(array('src' => 'media/images/flags-langes.png'). array ('title' => 'Spanish', 'class' => 'img-responsive', 'alt' => 'Spanish flag')))/;?>
```

Comment:
image path is broken 'media/images/flags-langes.png' should be 'media/images/flags-lang/es.png'

Reply:
It should be fixed now.